plugins/system/sef/src/Extension/Sef.php

```diff
@@ -68,7 +68,7 @@ public function onAfterInitialise()
         if (
             $app->get('sef')
             && !$app->get('sef_suffix')
-            && $this->params->get('trailingslash')
+            && $this->params->get('trailingslash', -1) !== -1
```

Comment:
Hmm, elsewhere we either use non-strict comparison, or we cast it to int when using strict comparison.
```suggestion
            && $this->params->get('trailingslash', -1) != -1
```